md-dao/src/main/java/com/wipro/ats/bdre/md/dao/UserRolesDAO.java

```diff
@@ -49,6 +52,28 @@
         return userRoles;
     }
 
+    public Map<Integer,String> diffRoleList() {
+        Object ob[];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1197)